bin/pin-github-actions.js

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env node
+
+import glob from 'fast-glob'
+import { fileURLToPath } from 'url'
+import path from 'node:path'
+import utils from 'util'
+import { exec } from 'child_process'
+
+export const execute = utils.promisify(exec);
+
+const __filename = fileURLToPath(import.meta.url);
+const __dirname = path.dirname(__filename);
+
+const args = process.argv;
+if (args.length !== 3) {
+  console.log(`Usage: bin/${path.basename(__filename)} <GITHUB_ACCESS_TOKEN>.\n\nThis script needs a Github access token to avoid hitting rate limits.\nYou can create one at https://github.com/settings/tokens.`)
```

Comment:
You can also detect if it's a Shopifolk and get it automatically by running `dev github print-auth`.